dist/extensions/AdvancedSearch/i18n/be.json

```diff
@@ -3,7 +3,8 @@
 		"authors": [
 			"Artsiom91",
 			"Da voli",
-			"Maksim L."
+			"Maksim L.",
```

Comment:
This looks like i18n changes to unrelated extensions which makes this PR hard to overview, do we have to commit these to make CI happy (will it complain about sync.sh run being required)?

Reply:
In order to remove the extension I thought the cleanest approach would be to remove it from the wikiman.yaml and sync everything, but because a general sync seems to be required in general (which I also did in this PR) that should probably happen in a different PR beforehand

Reply:
I tried to make it easier to look at with different commits but a different PR is probably better

Reply:
I wonder if this would not happen now after https://github.com/wbstack/mediawiki/pull/224 has been merged.